ansible/templates/usb/scripts/computenode.sh.j2

```diff
@@ -80,6 +80,11 @@ if [[ -f /var/lib/setup.json ]]; then
 fi
 
 function configure_erigonesd() {
+	if [[ "${CONFIG_remote_node}" == "1" ]]; then
+		# if this is a remote node, connect always to one IP that was specified during install at cfgdb connect
+		# (this IP is expected to be a proxy for all services)
+		MGMT_IP="${CFGDB_IP}"
```

Comment:
Lets discuss this first.
We should first query the CFGDB for a remote IP (e.g. `/esdc/settings/rabbitmq/remote-host`) and then gradually fallback to CFGDB_IP.
Similar for MON_IP...

Reply:
Also, we should not modify the MGMT_IP global variable, instead we should introduce another variable and set it in `query_cfgdb()`.